### PR TITLE
Added windows and bumped to 3.5.1

### PR DESCRIPTION
**Pull request by pariterre**

### CODE REVIEW

**Comment by pariterre** [comment]
@conda-forge-admin please rerender

**Comment by pariterre** [comment]
@conda-forge-admin please rerender

**Comment by pariterre** [comment]
@conda-forge/core
Hello! AppVeyor is failing because of a SSL issue, do you know how to properly add AppVeyor (since Azur cannot work for Windows)?
Thank!

**Comment by scopatz** [comment]
Appveyor isn't needed if you can get Azure to pass

**Comment by pariterre** [comment]
@conda-forge-admin please rerender

**Comment by pariterre** [comment]
@scopatz 
Yep, I did not know that Azure was working now. I just noticed it in another project. 

However, I don't understand the failing message in Azure for Windows.. Do you have any clue?

Thanks a lot

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,18 +8,19 @@ package:
...
-  skip: true  # [np==111 and np==112]
+  number: 0
+  skip: True  # [win and vc<14]
+  skip: True  # [py<35 or py>37]
   features:
 
 requirements:
   build:
-    - pkg-config  # [osx]
+    - pkgconfig
```
Comment (scopatz) [inline recipe/meta.yaml]: This seems like it is missing a dash

**Comment by pariterre** [comment]
@scopatz 
Thanks for your help with Azure and AppVeyor!
I have one remaining problem:
This is what I think is the problem message from the AppVeyor output:
```
INFO:conda_build.build:Packaging casadi
Packaging casadi
INFO conda_build.build:build(1571): Packaging casadi
INFO:conda_build.build:Packaging casadi-3.5.1-py36hfcfab1c_0
Packaging casadi-3.5.1-py36hfcfab1c_0
INFO conda_build.build:bundle_conda(891): Packaging casadi-3.5.1-py36hfcfab1c_0
No files or script found for output casadi
WARNING:conda_build.build:No files or script found for output casadi
WARNING conda_build.build:bundle_conda(971): No files or script found for output casadi
number of files: 0
```
Since I am using the same meta file for UNIX and Windows, I don't understand why in one case, it finds files and on the other one it doesn't... Any suggestion?

Thanks again!

File: recipe/blt.bat [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
+mkdir build
+cd build
+
+cmake ../^
+    -GNinja^
+    -DCMAKE_BUILD_TYPE=Release^
+    -DCMAKE_INSTALL_PREFIX="%PREFIX%"^
```
Comment (isuruf) [inline recipe/blt.bat]:
```suggestion
    -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%"^
```

**Comment by isuruf** [comment]
File name should be `bld.bat` instead of `blt.bat`

**Comment by pariterre** [comment]
> File name should be `bld.bat` instead of `blt.bat`

Wow.. I actually double-check the name, I was probably sleeping while doing so. Sorry about that... It works now (I can solve the new problems :P )

**Comment by pariterre** [comment]
Windows is actually failing when compiling casadi via the source, they probably have a CMakeLists problem. We should wait for the next versions until they solve the problem, then remove the skip. Everything should work at that time.

@pstjohn you can merge! You can give me admin access if you want me to take it from here!

**Comment by pstjohn** [comment]
Thanks for this! I unfortunately haven't had much time to devote to casadi packaging recently, so thanks for putting in the effort. You're welcome to add yourself to the recipe-maintainers section of the meta.yaml, I think that the best way to grant admin access.

**Comment by scopatz** [comment]
This LGTM

**Comment by pariterre** [comment]
Ready to merge!

**Pull request merged**